.gitignore

```diff
@@ -8,7 +8,6 @@
 /examples/sample-app/logs/openshift.log
 *.swp
 .vimrc
-.kubeconfig
```

Comment:
No, but there was a comment LOOONG ago when I was trying to *add* a gitignore for openshift config that since we don't look in pwd for config any more, there was no reason for this cruft. Doesn't matter much to me... you want to keep this?